wren-ai-service/src/pipelines/generation/question_rephrasing.py

```diff
@@ -0,0 +1,160 @@
+import logging
+import sys
+from datetime import datetime
+from typing import Any, Optional
+
+import orjson
+from hamilton import base
+from hamilton.async_driver import AsyncDriver
+from haystack.components.builders.prompt_builder import PromptBuilder
+from langfuse.decorators import observe
+from pydantic import BaseModel
+
+from src.core.pipeline import BasicPipeline
+from src.core.provider import LLMProvider
+
+logger = logging.getLogger("wren-ai-service")
+
+_system_prompt = """
+### TASK
+
+You are an expert at rephrasing questions to be more clear and specific while maintaining the original intent. You have access to the database schema which helps you understand the available data and relationships.
+
+### INSTRUCTIONS
+
+1. Read the user's question and understand the core intent
+2. Review the provided database schema to understand available data
+3. Rephrase the question to be more clear and specific, using proper table and column names from the schema where relevant
+4. Ensure the rephrased question aligns with the actual data structure
+5. Maintain the original meaning and requirements
+6. Use natural, conversational language
+7. Keep the rephrased question concise
+8. Avoid referencing tables or columns that don't exist in the schema
+
+### OUTPUT FORMAT
+
+Return the rephrased question as a simple string that accurately reflects both the user's intent and the database structure.
```

Comment:
I suggest still mentioning output format and telling llm to generate JSON here. Since not llms support structured output as of now